workflow/controller/operator.go

```diff
@@ -1719,6 +1725,20 @@ func getChildNodeIndex(node *wfv1.NodeStatus, nodes wfv1.Nodes, index int) *wfv1
 	return &lastChildNode
 }
 
+func getChildNodeIdsRetried(node *wfv1.NodeStatus, nodes wfv1.Nodes) []string {
+	childrenIds := []string{}
+	for i := 0; i < len(node.Children); i++ {
+		n := getChildNodeIndex(node, nodes, i)
+		if n == nil {
+			continue
```

Comment:
curious - what could cause us to hit the "continue" line? Is it just sort of "in case"?

Reply:
Yes, just in case.